common/client/versionChecker.go

```diff
@@ -56,6 +56,8 @@ const (
 	GoWorkerConsistentQueryVersion = "1.5.0"
 	// JavaWorkerRawHistoryQueryVersion indicates the minimum client version of the java worker which supports RawHistoryQuery
 	JavaWorkerRawHistoryQueryVersion = "1.3.0"
+	// JavaWorkerConsistentQueryVersion indicates the minimum client version of the java worker which supports RawHistoryQuery
+	JavaWorkerConsistentQueryVersion = "1.4.0"
```

Comment:
`1.4.0` is this the release version? If so we should probably target `3.2.0`

Reply:
Ahh, never mind its a FEATURE_VERSION. We should probably bump it though?

Reply:
Yep bumped it to 1.5